src/character.cpp

```diff
@@ -793,7 +793,12 @@ int Character::weight_capacity() const
 
 int Character::volume_capacity() const
 {
-    int ret = 0;
+    return volume_capacity_reduced_by( 0 );
+}
+
+int Character::volume_capacity_reduced_by( int mod ) const
```

Comment:
> Volume recalculation was fixed (it's not linear because of the PACKMULE and DISORGANIZED traits. The previous code didn't take that into account). This will be useful for recalculating volumes without cloning g->u (in a further PR)

The cleanest solution of the problem would look somewhat like [this](https://github.com/CleverRaven/Cataclysm-DDA/pull/16923#issuecomment-222509334), but it's out of scope for now. I've chosen the easiest way.